### PR TITLE
Camera console micro-optimization

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

The code is stupid and was overtiming slightly. No longer will it be stupid.

## Why It's Good For The Game

Game go zoom zoom

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image
 

</details>

## Changelog
:cl:
code: Slightly optimized and improved camera console camera detection code.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/camera.dm [hunk 1]
```diff
@@ -201,23 +201,17 @@
 
 // Returns the list of cameras accessible from this computer
 /obj/machinery/computer/security/proc/get_available_cameras()
-	var/list/L = list()
-	for (var/obj/machinery/camera/C in GLOB.cameranet.cameras)
+	var/list/camlist = list()
+	for(var/obj/machinery/camera/cam in GLOB.cameranet.cameras)
```
Comment (ike709) [inline code/game/machinery/computer/camera.dm]:
```suggestion
	for(var/obj/machinery/camera/cam as() in GLOB.cameranet.cameras)
```

File: code/modules/modular_computers/file_system/programs/secureye.dm [hunk 2]
```diff
@@ -176,20 +176,14 @@
 
 // Returns the list of cameras accessible from this computer
 /datum/computer_file/program/secureye/proc/get_available_cameras()
-	var/list/L = list()
-	for (var/obj/machinery/camera/cam in GLOB.cameranet.cameras)
+	var/list/camlist = list()
+	for(var/obj/machinery/camera/cam in GLOB.cameranet.cameras)
```
Comment (ike709) [inline code/modules/modular_computers/file_system/programs/secureye.dm]:
```suggestion
	for(var/obj/machinery/camera/cam as() in GLOB.cameranet.cameras)
```

File: code/game/machinery/computer/camera.dm (hunk 1 above)
Comment (itsmeow) [reply code/game/machinery/computer/camera.dm]: are we sure the camera is not null, because it could be qdeleted

**Comment by itsmeow** [comment]
wait how the hell did this even work when it fails lint check

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**